services/user.py

```diff
@@ -0,0 +1,48 @@
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> User:
+    user = User.objects.create_user(username=username,
+                                    password=password,
+                                    )

```

Comment:
You're calling `User.objects.create_user()` with just the `username` and `password` parameters. This method can also accept `email`, `first_name`, and `last_name` as parameters. You can pass these parameters directly instead of setting them later. This way you can avoid calling the `save()` method multiple times, which is against the DRY (Don't Repeat Yourself) principle.